diff --git a/dist/index.cjs b/dist/index.cjs
index 2921cb6d91fec0227d60c403c4cb7b1297c0f2e3..954efa909441ed4f16eb96ba60d7b07cfb93d46a 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -66,11 +66,7 @@ function vitePreprocessor(userConfig) {
           name: filenameWithoutExtension
         },
         rollupOptions: {
-          output: {
-            // override any manualChunks from the user config because they don't work with UMD
-            // eslint-disable-next-line @typescript-eslint/no-explicit-any
-            manualChunks: false
-          }
+          output: {}
         }
       }
     };
diff --git a/dist/index.mjs b/dist/index.mjs
index 88981fa66db5fc6f8d1bff495391ea71477aeff7..cf0e47eaeabcc4756a2a40929892d11163027c26 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -56,11 +56,7 @@ function vitePreprocessor(userConfig) {
           name: filenameWithoutExtension
         },
         rollupOptions: {
-          output: {
-            // override any manualChunks from the user config because they don't work with UMD
-            // eslint-disable-next-line @typescript-eslint/no-explicit-any
-            manualChunks: false
-          }
+          output: {}
         }
       }
     };
